src/components/views/home/index.jsx

```diff
@@ -1,20 +1,19 @@
 import * as React from 'react';
-import { View, Text, Button } from 'react-native';
+import { View, Text } from 'react-native';
 import { useTranslation } from 'react-i18next';
 
-import { styles } from '../views.styles';
+import LinkButton from 'components/molecules/link-button';
 
 export const HomeScreen = ({ navigation }) => {
   const { t } = useTranslation();
-
   return (
-    <View style={styles.container}>
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
```

Comment:
@andreaforni why do you want to remove shared styles that can be reused for all view and hardcode it inside this view's file ?

Reply:
I don't, I've only removed the old `index.JSX` and renamed the `index.JS` into JSX. I didn't touch its content.

But thanks for finding the issue, I've updated the file.